README.md

```diff
@@ -1,5 +1,19 @@
 # MOD.UK Frontend
 
+**20 October 2023:** The Defence Developer Services (D2S) will shortly commence
+work on its re-use strategy and the discovery and delivery of a re-use
+components service. As part of this, we will commence a Discovery activity for
+the Design System Toolkit. During the Discovery Phase, the MOD.UK Frontend and
+Design Library will be in maintenance mode with no further enhancements being
+made during this time. Only high priority issues such as security fixes or
+outages will be completed.
+
+**Getting support**\
```

Comment:
Nitpick: strictly speaking, this should be a heading, but I think I can see why it isn't.

Reply:
Talked to Rachael and this sentence should be removed